src/Microsoft.PowerShell.Commands.Management/commands/management/Service.cs

```diff
@@ -1413,7 +1515,7 @@ public ServiceStartMode StartupType
         }
         // We set the initial value to an invalid value so that we can
         // distinguish when this is and is not set.
-        internal ServiceStartMode startupType = (ServiceStartMode)(-1);
+        internal ServiceStartupType startupType = (ServiceStartupType)(-1);
```

Comment:
Sorry, I just noticed this. Your use of a wrapper enum + conversion functions makes it possible to define -1 as an ServiceStartupType value so it does not need to be casted throughout. Something like ServiceStartupType.InvalidValue or ServiceStartupType.Undefined would work.